Sage/carbon
# chore(playwright): remove generated bundle when checking out a branch

### Proposed behaviour

- Add script to remove bundle generated by Playwright, `playwright/.cache`, when switching/checking out a git branch

### Current behaviour

- Since `playwright/.cache` is ignored by git, so the directory will not be removed when switching between branches. Since this directory is an important artifact used for Playwright's Component tests, not removing it after test runs will mean code from previously-visited branches will remain in the bundle, which could result in errors.

### Checklist




#### QA


### Testing instructions

In the `carbon` directory, run:

```sh
mkdir playwright-report
mkdir playwright/.cache
touch playwright/.cache/asset.js
```

then switch to an arbitrary branch, then back to this PR's branch.

**Verify that the directories `playwright/.cache` and `playwright-report` are removed after doing this.**
